zerver/views/__init__.py

```diff
@@ -1307,3 +1316,15 @@ def email_unsubscribe(request, type, token):
 
     return render_to_response('zerver/unsubscribe_link_error.html', {},
                               context_instance=RequestContext(request))
+
+@login_required(login_url = settings.HOME_NOT_LOGGED_IN)
+def mobile_redirect(request):
+    user_profile = request.user
+    params = {
+        "api_key": user_profile.api_key,
+        "email": user_profile.email,
+        "domain": settings.EXTERNAL_HOST
+    }
+    msg = "%s:///?%s" % ("zulip", urlencode(params))
+    page = "<meta HTTP-EQUIV='REFRESH' content='0; url=%s'>" % (msg)
```

Comment:
Please use `% (msg,)` ;  I guess I should also figure out why lint didn't catch this.